xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -202,6 +202,10 @@ enum AppAction {
     case indexOurSphere
     case succeedIndexOurSphere(OurSphereRecord)
     case failIndexOurSphere(String)
+    /// Developer utility function, for now
+    case resetIndex
+    case succeedresetIndex
+    case failresetIndex(String)
```

Comment:
This seems like an unrelated change.

Also a potential footgun w/r/t syncing database with sphere which is currently pretty carefully made to be atomic. I haven't worked out all the implications, so could be wrong about that.

Reply:
> This seems like an unrelated change.

It's near impossible to test excerpt generation without it and it's a developer utility... I don't see how it's much different to resetting the first run experience? 

We know the indexing process is designed to read the entire sphere if the `since` field is NULL, which is all this does.